chore(prettier): move prettier packages to dev dependencies

trybick opened the pull request:
#### Summary

Resolves: 

- Install prettier package. Eslint and prettier were giving errors in VSCode Output.

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in semantic format `TYPE(scope): description`
- [x] Commits Messages are in semantic format `TYPE(scope): description`
- [x] Has Summary
- [x] Has Labels

trybick committed:
chore(app): Install prettier package

davidholyko reviewed (changes requested):
revert package-lock deletion. i expect a new package lock to replace the old one though

davidholyko commented on package.json:
```diff
@@ -50,7 +49,9 @@
   },
   "devDependencies": {
     "eslint": "^6.6.0",
+    "eslint-plugin-prettier": "^3.2.0",
```
I know the exact difference between `devDependencies` and `dependencies`. Can you explain the why this was move to `devDependencies`

trybick replied:
Prettier and eslint aren't used in production, so that's why dev dependencies. Build time will be shorter, less time for netlify to install packages.

davidholyko commented:
I think we can add `prettier` to our list of scopes from this PR.

https://github.com/f1v/full-git-commit-history-app/blob/master/.github/semantic.yml#L28-L37

trybick committed:
add prettier scope

trybick committed:
Remove package lock and reinstall modules

trybick commented:
> revert package-lock deletion. i expect a new package lock to replace the old one though

I just wiped out package lock and node_modules and it's still saying it's mostly deleted. I don't get it but I think it's ok.

davidholyko commented:
Probably do

```
npm i
```

and add / commit / push the newly generated package-lock file

trybick commented:
That is what I did. But no new diffs are generated. 🙃

davidholyko commented:
ooo that is strange. Ill take a closer look at this

davidholyko commented:
I got this npm error on your branch after 

```
rm -rf node_modules
rm package-lock.json
npm i
```

do you know how to resolve this

<img width="600" alt="Screen Shot 2020-12-13 at 1 40 39 PM" src="https://user-images.githubusercontent.com/20917792/102020679-cfe89d00-3d48-11eb-8f6e-e6767dd15ed3.png">

trybick committed:
checkout master package-lock and reinstall from there

trybick committed:
Merge branch 'master' into fix/prettier/install-prettier

trybick committed:
Merge branch 'master' into fix/prettier/install-prettier

trybick commented:
I am not sure honestly (I wasn't getting that error). But previously I had deleted and package lock and npm i which generated a lot of diffs. I reverted that, checked out master package lock, then `npm i` which should just pick up the difference from the new packages. Hopefully that solves it.

davidholyko commented:
~looking at the diffs is still pretty strange. it might be a github caching issue. Do you mind if we close this PR and make a new one in a new branch with all of the changes~

nevermind this. :)

davidholyko commented on .github/semantic.yml:
```diff
@@ -32,3 +32,4 @@ scopes:
   - revert
   - tests
   - webpack
+  - prettier
```
this now redundant that its in this file twice. we can remove the diff in this file

https://github.com/f1v/full-git-commit-history-app/blob/53da35c00f49da0c7681933a6c7d3fe9d698832a/.github/semantic.yml#L30-L35

trybick commented:
Closing, opening new PR #10
